pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/BaseMultipleSegmentsConversionExecutor.java

```diff
@@ -206,6 +232,8 @@ public List<SegmentConversionResult> executeTask(PinotTaskConfig pinotTaskConfig
         File convertedTarredSegmentFile = tarredSegmentFiles.get(i);
         SegmentConversionResult segmentConversionResult = segmentConversionResults.get(i);
         String resultSegmentName = segmentConversionResult.getSegmentName();
+        _eventObserver.notifyProgress(_pinotTaskConfig,
+            String.format("Uploading segment: %s, %d/%d", resultSegmentName, count++, numOutputSegments));
```

Comment:
Using `count` here seems incorrect. It'd be good to use `i + 1`?

Reply:
sharp eye. thx